pkg/consts/consts.go

```diff
@@ -2,6 +2,7 @@ package consts
 
 const (
 	OperatorOldestSupportedVersion = "v2.2.0"
+	ClusterOperatorLastVersion     = "v2.4.4"
```

Comment:
m i right this means they are only able to use the function if they have upgraded the old cluster to the latest?

Reply:
I'm not sure I understand the question, but to clarify: 
We had a function to dynamically discover the old cluster-operator "last version" (i.e the latest release). But now that cluster-operator is no longer active, the "last version" is static at v2.4.4. 

This version is only used to check if the installed operator (discovered via image tag) is lower than this. If it is, we use that installed operator version to then pull the manifests for the uninstall process.